gradle/libs.versions.toml

```diff
@@ -2,14 +2,15 @@
 autoService = "1.0"
 gjf = "1.11.0"
 jvmTarget = "1.8"
-kotlin = "1.6.0"
+kotlin = "1.6.10"
+dokka = "1.6.10"
 kotlinCompileTesting = "1.4.6"
 kotlinpoet = "1.10.2"
 ksp = "1.6.0-1.0.1"
```

Comment:
Should this be updated to `1.6.10-1.0.1` or even `1.6.10-1.0.2` to be in-sync with `kotlin`?

Reply:
In a separate PR